6.2/stay_positive.py

```diff
@@ -3,7 +3,7 @@ def get_positive_numbers() -> list[int]:
     Get from the user list of numbers and return a list of the positive once after changing them into integers.
     :return: List's object of the positive numbers that the user inserted after changing them into integers.
     """
-    return list(map(int, list(filter(lambda number: number >= 0, get_input_from_user()))))
+    return [int(number) for number in get_input_from_user() if number >= 0]
```

Comment:
In this assignment it wasn't clear, but in general we should Seperate IO from logic unless there is a good reason to do so.
https://wiki.c2.com/?SeparateIoFromCalculation